bluepyopt/ephys/locations.py

```diff
@@ -133,7 +133,7 @@ def instantiate(self, sim=None, icell=None):
     def __str__(self):
         """String representation"""
 
-        return '%s[%s]' % (self.seclist_name, self.sec_index)
+        return '%s: %s[%s]' % (self.name, self.seclist_name, self.sec_index)
```

Comment:
well, the only information contained in this location is the sec_list and sec_index. It doesn't matter if it's a different object setting, the user has all the information necessary